docs/pywbemcligeneraloptions.rst

```diff
@@ -21,15 +21,13 @@ information on each command-group arguments and options):
   * Port: (optional) defines WBEM server port to be used [Defaults: 5988(HTTP)
     and 5989(HTTPS)]. (EnvVar: PYWBEMCLI_SERVER).
 
-  The server parameter is conditionally optional (see ``--name`` options). If the
-  ``--name`` option exists and there is a server with the name defined by
-  ``--name`` defined in the :term:`connections file` the parameters of that
-  name are used for the connection.
+  The server parameter is mutually exclusive with the ``--name`` option and the
+  ``mock-server option since each define a connection to a WBEM server.
```

Comment:
"``mock-server" misses the trailing backquotes and the leading double dash.

Reply:
DONE